lib/rspec/matchers/dsl.rb

```diff
@@ -358,6 +373,9 @@ class Matcher
         # object wrapped by `expect`.
         attr_reader :actual
 
+        # @private
+        attr_accessor :actual_error
```

Comment:
Is it necessary to expose `actual_error` as an instance variable and attribute method?  Can't it just be a local?